config/config.php

```diff
@@ -81,13 +81,21 @@
 	'production' => [
 	],
 	'stable' => [
+		'10.9' => [
+			'latest' => '10.9.1',
+			'web' => 'https://doc.owncloud.com/server/10.9/admin_manual/maintenance/upgrading/update.html',
+		],
+		'10.8' => [
+			'latest' => '10.9.1',
+			'web' => 'https://doc.owncloud.com/server/10.8/admin_manual/maintenance/upgrading/update.html',
+		],
 	],
 	'beta' => [
 	],
 	'daily' => [
 		'10.8' => [
```

Comment:
10.8 should be updated to 10.9.1 and not to daily master
10.9 should be updated to 10.9.1 as well
10.9.1 should be updated to daily master instead